numba_cuda/numba/cuda/tests/cudadrv/test_nvjitlink.py

```diff
@@ -251,5 +253,61 @@ def kernel():
                 pass
 
 
+class TestLinkerUsage(CUDATestCase):
```

Comment:
These probably need to be skipped if pynvjitlink isn't installed

Reply:
Fixed in https://github.com/NVIDIA/numba-cuda/pull/91/commits/38238bc6bba2288064a6c30fb5e368b085554921